lnd_test.go

```diff
@@ -4878,6 +4946,28 @@ func testInvoiceRoutingHints(net *lntest.NetworkHarness, t *harnessTest) {
 	// is offline.
 	ctxt, _ = context.WithTimeout(ctxb, channelCloseTimeout)
 	closeChannelAndAssert(ctxt, t, net, net.Alice, chanPointEve, true)
+
+	// Wait for the channel to be marked pending force close.
```

Comment:
Is this being done for every force close? In that case, we should just include this within `closeChannelAndAssert` to avoid all the code duplication.

Reply:
Not being done the same way always, as we sometime follow a different flow to assert certain behaviour after a force close.

Reply:
Gave this some thought, and decided to extract the common logic into a new method that can be called where the flow is similar. See https://github.com/lightningnetwork/lnd/pull/2153